tests/pipelines/stable_diffusion/test_fabric.py

```diff
@@ -0,0 +1,241 @@
+            "num_images": 1,
+            "num_inference_steps": 2,
+            "output_type": "np",
+            "height": 128,
+            "width": 128,
+        }
+        return inputs
+
+    def test_fabric(self):
+        device = "cpu"  # ensure determinism for the device-dependent torch.Generator
+
+        components = self.get_dummy_components()
+        pipe = FabricPipeline(**components)
+        pipe = pipe.to(device)
+
+        pipe.set_progress_bar_config(disable=True)
+
+        inputs = self.get_dummy_inputs(device)
+        output = pipe(**inputs)
+        image = output.images
+        image_slice = image[0, -3:, -3:, -1]
+        assert image.shape == (1, 128, 128, 3)
+        expected_slice = np.array(
+            [0.46241423, 0.45808375, 0.4768011, 0.48806447, 0.46090087, 0.5161956, 0.52250206, 0.50051796, 0.4663524]
+        )
+
+        self.assertTrue(np.allclose(image_slice.flatten(), expected_slice, atol=1e-2))
+
+    def test_fabric_w_fb(self):
+        device = "cpu"  # ensure determinism for the device-dependent torch.Generator
+
+        components = self.get_dummy_components()
+        pipe = FabricPipeline(**components)
+        pipe = pipe.to(device)
+
+        pipe.set_progress_bar_config(disable=True)
+
+        inputs = self.get_dummy_inputs(device)
+        inputs["liked"] = [Image.fromarray(np.ones((512, 512)))]
+        output = pipe(**inputs)
+        image = output.images
+        image_slice = output.images[0, -3:, -3:, -1]
+
+        assert image.shape == (1, 128, 128, 3)
+        expected_slice = np.array(
+            [
+                [0.46259943, 0.45826188, 0.4768875],
+                [0.4880805, 0.46087098, 0.5162324],
+                [0.5224824, 0.5005106, 0.46634308],
+            ]
+        ).flatten()
+
+        self.assertTrue(np.allclose(image_slice.flatten(), expected_slice, atol=1e-2))
+
+    def test_monkey_patching(self):
+        # Create a sample model and module
+        device = "cpu"
+        torch.manual_seed(0)
+        z_all = torch.randn(2, 4, 64, 64)
+        t = 0
+        prompt_embd = torch.randn(2, 77, 32)
+        cached_pos_hiddens = [torch.randn(1, 1024, 64)] * 6
+        print(len(cached_pos_hiddens))
+        self.get_dummy_inputs(device)
+        print(cached_pos_hiddens[0].shape)
+        # out = model.unet(z_all, t, encoder_hidden_states=prompt_embd)
+        components = self.get_dummy_components()
+        pipe = FabricPipeline(**components)
+        pipe = pipe.to(device)
+        pipeline = pipe.unet_forward_with_cached_hidden_states(
+            z_all, t, prompt_embd, cached_pos_hiddens=cached_pos_hiddens
+        )[0]
+
+        image_slice = pipeline[0, -3:, -3:, -1].detach().numpy()
+        expected_slice = np.array(
+            [[-0.0590, 0.3149, -0.1035], [0.0016, -0.1665, 0.1026], [-0.0626, 0.0607, -0.1045]]
+        ).flatten()
+
+        self.assertTrue(np.allclose(image_slice.flatten(), expected_slice, atol=1e-2))
+
+
+@nightly
```

Comment:
Okay so, I had a quick thought. Let's maybe remove the nightly test for now. Based on the usage of the pipeline we can always revisit this and add. This is improve our internal nightly test suit helping us run the most important pipelines in a faster manner. 

Does that make sense?

Reply:
makes sense. should I comment it all out or just remove it all?

Reply:
Removal makes sense.